### PR TITLE
Fix dates in August

### PR DESCRIPTION
**Pull request by elem15**

### CODE REVIEW

File: stage2/README.md [hunk 1]
```diff
@@ -186,7 +186,7 @@ At the end of this week, all students will be required to use typescript and adh
   - [Interviewing Students for CoreJS #1](https://github.com/rolling-scopes-school/tasks/blob/master/tasks/interview-basic-coreJS.md)
 </details>
 
-## Неделя #11. 31/01/2023
+## Неделя #11. 31/07/2023
```
Comment (valerydluski) [inline stage2/README.md:189]: 01/08/2023

File: stage2/README.md [hunk 2]
```diff
@@ -200,7 +200,7 @@ At the end of this week, all students will be required to use typescript and adh
   
 </details>
 
-## Неделя #12. 07/02/2023
+## Неделя #12. 07/08/2023
```
Comment (valerydluski) [inline stage2/README.md:203]: 08/08/2023

File: stage2/README.md [hunk 3]
```diff
@@ -225,7 +225,7 @@ At the end of this week, all students will be required to use typescript and adh
   - [Собеседуем студентов по CoreJS #2](https://github.com/rolling-scopes-school/tasks/blob/master/tasks/interview-corejs.md)
 </details>
 
-## Неделя #14. 21/02/2023
+## Неделя #14. 21/08/2023
```
Comment (valerydluski) [inline stage2/README.md:228]: 22/08/2023

File: stage2/README.md [hunk 4]
```diff
@@ -236,7 +236,7 @@ At the end of this week, all students will be required to use typescript and adh
   - [Собеседуем студентов по CoreJS #2](https://github.com/rolling-scopes-school/tasks/blob/master/tasks/interview-corejs.md)
 </details>
 
-## Неделя #15. 28/02/2023
+## Неделя #15. 28/08/2023
```
Comment (valerydluski) [inline stage2/README.md:239]: 29/08/2023

**Comment by valerydluski** [comment]
@elem15 Do you plan to respond to the comments?

**Comment by elem15** [comment]
Should I do something?

**Comment by valerydluski** [comment]
Please take a look at my comments on your changes. @elem15

**Review by valerydluski — APPROVED** [review]

**Comment by valerydluski** [comment]
@elem15 Please resolve the conflicts, and then we can proceed with the merge.

**Review by WiiJoy — APPROVED** [review]

**Pull request merged**